### ISSUE TITLE
how can I get the pictures labels

### ISSUE THREAD

**Issue by ke-s** [GuYuc/WS-DAN.PyTorch#29]
When I did model validation, I got three pictures (raw.jpg,raw_atten.jpg,heat_atten.jpg) I want to get the class label corresponding to each picture in the validation set. What should I do?
